include/vkgcDefs.h

```diff
@@ -86,6 +86,7 @@
 //  %Version History
 //  | %Version | Change Description                                                                                    |
 //  | -------- | ----------------------------------------------------------------------------------------------------- |
+//  |     52.3 | Add fastMathFlags to PipelineShaderOptions                                                            |
```

Comment:
I think you need to bump this: LLPC_INTERFACE_MINOR_VERSION

Reply:
Right, thanks